opensrp-reveal/src/main/java/org/smartregister/reveal/model/BaseCardDetails.java

```diff
@@ -0,0 +1,48 @@
+package org.smartregister.reveal.model;
+
+/**
+ * Created by samuelgithengi on 3/22/19.
+ */
+public class BaseCardDetails {
```

Comment:
Master already uses `CardDetails` as a base class for `SprayCardDetails` and `MosquitoCollectionCardDetails`.

Maybe resolve the differences?